lib/resources/r4/service_request.yaml

```diff
@@ -0,0 +1,118 @@
+---
+  name: ServiceRequest
+  field_name_base_url: https://www.hl7.org/fhir/servicerequest-definitions.html#ServiceRequest
+  fields:
+  - name: status
+    required: 'Yes'
+    type: code
+    description: The status of a service order.
+    binding:
+      description: The status of a service order.
+      terminology:
+      - display: Request Status
+        system: http://hl7.org/fhir/request-status
+        info_link: https://www.hl7.org/fhir/valueset-request-status.html
+  - name: intent
+    required: 'Yes'
+    type: code
+    description: The kind of service request.
+    binding:
+      description: The kind of service request.
+      terminology:
+      - display: Request Intent
+        system: http://hl7.org/fhir/request-intent
+        info_link: https://www.hl7.org/fhir/valueset-request-intent.html
+  - name: category
+    required: 'No'
+    type:  List of CodeableConcept
+    description: Classification of the requested service.
+    binding:
+      description: A code that classifies the request for service.
+      terminology:
+      - display: Service Request Category Codes
+        system: http://snomed.info/sct
+        info_link: https://www.hl7.org/fhir/valueset-servicerequest-category.html
```

Comment:
Had an offline discussion with Fran and it sounds like we would not like to reflect the support for SNOMED's until a code change on the local service has been made. Kindly remove the selected. Thanks!

Reply:
You can find the changes here: https://github.com/cerner/fhir.cerner.com/pull/389/commits/18a592885426eefd181f05aa020842939e654c6a